sdk/test-utils/test-utils-vitest/src/fakeTestSecrets.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Centralized location for test/fake secrets so that suppressions
+// are easier to maintain properly
+
+export const fakeTestPasswordPlaceholder = "p@55w0rd";
```

Comment:
need to add an entry to https://github.com/Azure/azure-sdk-for-js/blob/d6e1da0636f4f071dbf908abbe05feb86b381b32/eng/CredScanSuppression.json#L38

Reply:
Added entry